runbotics/runbotics-common/src/actions/actions.ts

```diff
@@ -214,6 +214,10 @@ export enum DesktopAction {
     PERFORM_KEYBOARD_SHORTCUT = 'desktop.performKeyboardShortcut',
 }
 
+export enum VisualBasicAction {
+    RUN_MACRO = 'vBasic.runMacro',
```

Comment:
write the complete name + imo we should call it script not macro `visualBasic.runScript`